test/com/google/javascript/jscomp/WhitespaceWrapGoogModulesTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.javascript.jscomp;
+
+import com.google.javascript.jscomp.CompilerOptions.LanguageMode;
+
+public class WhitespaceWrapGoogModulesTest extends CompilerTestCase {
+
+  private LanguageMode languageOut;
+
+  @Override
+  public void setUp() {
+    setAcceptedLanguage(LanguageMode.ECMASCRIPT6);
+    languageOut = LanguageMode.ECMASCRIPT6;
+    disableTypeCheck();
+    enableLineNumberCheck(false);
+    enableCompareAsTree(false); // otherwise the "use strict" pops to the top and breaks the test
```

Comment:
Here's what my test expects...
``` javascript
        "goog.loadModule(function(exports){"+
            "\"use strict\";"+
            "goog.module(\"test\");"+
            "var f=5;"+
            "exports=f;"+
            "return exports"+
        "})");
```
Test failure with that line removed:
```
Expected: goog.loadModule(function(exports){goog.module("test");var f=5;exports=f;return exports})
Result:   goog.loadModule(function(exports){"use strict";goog.module("test");var f=5;exports=f;return exports})
Node tree inequality:
Expected tree:
BLOCK [synthetic: 1]
    SCRIPT 1 [synthetic: 1] [source_file: expected0] [input_id: InputId: expected0]
        EXPR_RESULT 1 [source_file: expected0]
            CALL 1 [source_file: expected0]
                GETPROP 1 [source_file: expected0]
                    NAME goog 1 [source_file: expected0]
                    STRING loadModule 1 [source_file: expected0]
                FUNCTION  1 [source_file: expected0]
                    NAME  1 [source_file: expected0]
                    PARAM_LIST 1 [source_file: expected0]
                        NAME exports 1 [source_file: expected0]
                    BLOCK 1 [directives: [use strict]] [source_file: expected0]
                        EXPR_RESULT 1 [source_file: expected0]
                            CALL 1 [source_file: expected0]
                                GETPROP 1 [source_file: expected0]
                                    NAME goog 1 [source_file: expected0]
                                    STRING module 1 [source_file: expected0]
                                STRING test 1 [source_file: expected0]
                        VAR 1 [source_file: expected0]
                            NAME f 1 [source_file: expected0]
                                NUMBER 5.0 1 [source_file: expected0]
                        EXPR_RESULT 1 [source_file: expected0]
                            ASSIGN 1 [source_file: expected0]
                                NAME exports 1 [source_file: expected0]
                                NAME f 1 [source_file: expected0]
                        RETURN 1 [source_file: expected0]
                            NAME exports 1 [source_file: expected0]


Actual tree:
BLOCK [synthetic: 1]
    SCRIPT 1 [synthetic: 1] [source_file: testcode] [input_id: InputId: testcode]
        EXPR_RESULT 1 [source_file: testcode]
            CALL 1 [source_file: testcode]
                GETPROP 1 [source_file: testcode]
                    NAME goog 1 [source_file: testcode]
                    STRING loadModule 1 [source_file: testcode]
                FUNCTION  1 [source_file: testcode]
                    NAME  1 [source_file: testcode]
                    PARAM_LIST 1 [source_file: testcode]
                        NAME exports 1 [source_file: testcode]
                    BLOCK 1 [source_file: testcode]
                        EXPR_RESULT 1 [source_file: testcode]
                            STRING use strict 1 [source_file: testcode]
                        EXPR_RESULT 1 [source_file: testcode]
                            CALL 1 [source_file: testcode]
                                GETPROP 1 [source_file: testcode]
                                    NAME goog 1 [source_file: testcode]
                                    STRING module 1 [source_file: testcode]
                                STRING test 1 [source_file: testcode]
                        VAR 1 [source_file: testcode]
                            NAME f 1 [source_file: testcode]
                                NUMBER 5.0 1 [source_file: testcode]
                        EXPR_RESULT 1 [source_file: testcode]
                            ASSIGN 1 [source_file: testcode]
                                NAME exports 1 [source_file: testcode]
                                NAME f 1 [source_file: testcode]
                        RETURN 1 [source_file: testcode]
                            NAME exports 1 [source_file: testcode]


Expected subtree: BLOCK 1 [directives: [use strict]] [source_file: expected0]
    EXPR_RESULT 1 [source_file: expected0]
        CALL 1 [source_file: expected0]
            GETPROP 1 [source_file: expected0]
                NAME goog 1 [source_file: expected0]
                STRING module 1 [source_file: expected0]
            STRING test 1 [source_file: expected0]
    VAR 1 [source_file: expected0]
        NAME f 1 [source_file: expected0]
            NUMBER 5.0 1 [source_file: expected0]
    EXPR_RESULT 1 [source_file: expected0]
        ASSIGN 1 [source_file: expected0]
            NAME exports 1 [source_file: expected0]
            NAME f 1 [source_file: expected0]
    RETURN 1 [source_file: expected0]
        NAME exports 1 [source_file: expected0]


Actual subtree: BLOCK 1 [source_file: testcode]
    EXPR_RESULT 1 [source_file: testcode]
        STRING use strict 1 [source_file: testcode]
    EXPR_RESULT 1 [source_file: testcode]
        CALL 1 [source_file: testcode]
            GETPROP 1 [source_file: testcode]
                NAME goog 1 [source_file: testcode]
                STRING module 1 [source_file: testcode]
            STRING test 1 [source_file: testcode]
    VAR 1 [source_file: testcode]
        NAME f 1 [source_file: testcode]
            NUMBER 5.0 1 [source_file: testcode]
    EXPR_RESULT 1 [source_file: testcode]
        ASSIGN 1 [source_file: testcode]
            NAME exports 1 [source_file: testcode]
            NAME f 1 [source_file: testcode]
    RETURN 1 [source_file: testcode]
        NAME exports 1 [source_file: testcode]
```
It seems "use strict" is treated specially as a directive which breaks the comparison. So I changed the test to compare the source rather than the AST.